lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
@zachthompson let's do that for now so tests pass as expected. Then we can work towards a better approach after